src/PhpBrew/VariantBuilder.php

```diff
@@ -315,34 +315,27 @@ public function __construct()
             return;
         };
 
-        $this->variants['curl'] = function (Build $build, $prefix = null) {
-            if ($prefix) {
-                return "--with-curl=$prefix";
-            }
-
-            if ($prefix = Utils::findIncludePrefix('curl/curl.h')) {
-                return "--with-curl=$prefix";
-            }
+        $this->variants['curl'] = function (Build $build, $val = null) {
```

Comment:
Why is the argument changed from `$prefix` to `$val`?

Reply:
Please see the big comment below.